cli/cli/helpers/engine_manager/engine_existence_guarantor.go

```diff
@@ -224,7 +224,7 @@ func (guarantor *engineExistenceGuarantor) VisitRunning() error {
 			engineRestartCmd,
 		)
 	} else {
-		logrus.Debugf("Currently running engine version '%v' is >= the version the CLI expects", guarantor.maybeCurrentlyRunningEngineVersionTag)
+		logrus.Warningf("Currently running engine version '%v' is >= the version the CLI expects", guarantor.maybeCurrentlyRunningEngineVersionTag)
```

Comment:
@h4ck3rk3y this is now going to go out to users, so can we make it more user-friendly (e.g. making it a complete sentence; giving them information on how to resolve)

Reply:
Ha also saw that this condition would run if the versions are equal as well. Have put in a fix.